packages/styling/src/interfaces/IPalette.ts

```diff
@@ -231,4 +231,24 @@ export interface IPalette {
    * Color code for greenLight.
    */
   greenLight: string;
+
```

Comment:
Remove these high contrast colors. In the CSS, directly use these variables:
https://developer.mozilla.org/en-US/docs/Web/CSS/color_value#System_Colors